models/reviews.model.js

```diff
@@ -60,4 +60,20 @@ exports.updateReviewById = (newVotes, id) => {
       } 
     return rows
   })
+}
+
+exports.selectReviewComments = (id) => {
+  return db
+  .query(
+    `SELECT * FROM comments WHERE comments.review_id = $1;`, [id])
+  .then(({ rows }) => {
+    const comments = rows;
+    if (!comments) {
```

Comment:
ideally, if there are no comments, send a status 200 and an empty array ... we want access to the comments array even if it's empty